crates/matrix-sdk-base/src/client.rs

```diff
@@ -1270,20 +1147,17 @@ impl BaseClient {
         changes: &StateChanges,
         room_info_notable_updates: BTreeMap<OwnedRoomId, RoomInfoNotableUpdateReasons>,
     ) {
-        if changes.account_data.contains_key(&GlobalAccountDataEventType::IgnoredUserList) {
-            if let Some(event) =
-                changes.account_data.get(&GlobalAccountDataEventType::IgnoredUserList)
-            {
-                match event.deserialize_as::<IgnoredUserListEvent>() {
-                    Ok(event) => {
-                        let user_ids: Vec<String> =
-                            event.content.ignored_users.keys().map(|id| id.to_string()).collect();
-
-                        self.ignore_user_list_changes.set(user_ids);
-                    }
-                    Err(error) => {
-                        warn!("Failed to deserialize ignored user list event: {error}")
-                    }
+        if let Some(event) = changes.account_data.get(&GlobalAccountDataEventType::IgnoredUserList)
+        {
+            match event.deserialize_as::<IgnoredUserListEvent>() {
+                Ok(event) => {
+                    let user_ids: Vec<String> =
+                        event.content.ignored_users.keys().map(|id| id.to_string()).collect();
+
+                    self.ignore_user_list_changes.set(user_ids);
+                }
+                Err(error) => {
+                    warn!("Failed to deserialize ignored user list event: {error}")
```

Comment:
Should this be logged as an actual error?

Reply:
sure, why not